modules/node/src/appRegistry/appActions.service.ts

```diff
@@ -99,9 +103,13 @@ export class AppActionsService {
 
     // take action and uninstall
     this.log.log(`Unlocking transfer ${senderApp.identityHash}`);
-    await this.cfCoreService.takeAction(senderApp.identityHash, {
-      preImage: action.preImage,
-    } as SimpleLinkedTransferAppAction);
+    await this.cfCoreService.takeAction(
+      senderApp.identityHash,
+      {
+        preImage: action.preImage,
+      } as SimpleLinkedTransferAppAction,
+      LINKED_TRANSFER_STATE_TIMEOUT,
+    );
```

Comment:
We technically dont need to specify a timeout here because this is a unilateral action that theoretically should use the `defaultTimeout` automaticallly in `progressState`